roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_image }}:{{ aci_gw_version }}
```

Comment:
yes. most of the time we will be changing the tags and rarely will be changing the docker image name. That was the point having different variables here.